pages/glossary/focusable.md

```diff
@@ -8,4 +8,15 @@ input_aspects:
   - DOM tree
 ---
 
-Elements that can become the target of keyboard input as described in the [HTML](https://www.w3.org/TR/html) specification of [focusable](https://html.spec.whatwg.org/#focusable-area) and [can be focused](https://html.spec.whatwg.org/#specially-focusable).
+An element is _focusable_ if one or both of the following are true:
+
+- the element is part of [sequential focus navigation][]; or
+- the element has a [tabindex value][] that is not null.
+
+**Exception**: Elements that lose focus immediately after the focus event is fired are not considered _focusable_.
```

Comment:
What does "immediately" means? Less than 1s? 1ms?
If JS is used to redirect focus, there will necessarily be some time for the redirection to occur (and it might actually depend on the computer and its load).

Reply:
On the next tick is what we were thinking, just didn't quite know how to define that. This seemed reasonable... but perhaps not. Any suggestions?

Reply:
"next tick" might be too restrictive (not fully sure exactly what it is, however).
`<button onfocus="this.blur(); someElement.focus()">` would loose focuse on "next tick" (I guess), but
`<button onfocus="console.log('Redirecting…'); this.blur(); someElement.focus()">` wouldn't.

And even the first case, I'm not fully sure what a "tick" is, but simply calling a JS function can actually do a lot of things and may take several ticks.

Reply:
Going back to using a time limit (1 sec) without user intervention